src/components/Folder.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect, MouseEventHandler } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import Section from "./Section";
+import SortMenu from "./SortMenu";
+import Card from "./Card";
+import Search from "./Search";
+import * as S from "./Styled";
+import { ShowAll, FetchFolderData } from "../utils/Api";
+import { useFoldLink } from "./hooks/useFolder";
+import { ChangeNameModal, DeleteFolder, ShareModal } from "./modal/index";
+import shareImg from "../Image/share.svg";
+import penImg from "../Image/pen.svg";
+import deleteImg from "../Image/delete.svg";
+import { FolderData, LinkData } from "../utils/type";
+
+function Folder() {
+  const [selectSortName, setSelectSortName] = useState<number>(0);
+  const [foldLinkTitle, setFoldLinkTitle] = useState<string>("전체");
+  const [sortData, setSortData] = useState<FolderData[]>([]);
```

Comment:
지금도 좋지만 타입이 충분히 추론되는 경우에는 굳이 타입을 따로 적어주지 않아도 괜찮습니다.
useState에 0을 넘기면 해당 state가 자연스럽게 number 타입으로 추론되어요.
L20 처럼 단순히 배열 리터럴([]) 만으로 FolderData의 배열임이 추론되지 않는 경우에는 제네릭을 넣어주는 게 의미 있습니다.


```suggestion
  const [selectSortName, setSelectSortName] = useState(0);
  const [foldLinkTitle, setFoldLinkTitle] = useState("전체");
  const [sortData, setSortData] = useState<FolderData[]>([]);
```